### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by aledeg**
Changes proposed in this pull request:

- Add missing translation on the extension page

How to test the feature manually:

1. remove a system extension
2. remove a user extension

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: app/i18n/en/feedback.php [hunk 1]
```diff
@@ -47,6 +48,7 @@
 		'no_access' => 'You have no access on %s',
 		'not_enabled' => '%s is not enabled',
 		'not_found' => '%s does not exist',
+		'removed' => '%s deleted',
```
Comment (Frenzie) [inline app/i18n/en/feedback.php]: Remove might be better than delete? But either way since they're new I'd keep the key and the text the same.

Comment (aledeg) [reply app/i18n/en/feedback.php]: Sure, I can change that.
The keys are not new, they've never been translated.

Comment (aledeg) [reply app/i18n/en/feedback.php]: done

**Comment by aledeg** [comment]
I've added a quick method to be able to check if a key exists

File: app/i18n/en/feedback.php [hunk 2]
```diff
@@ -36,6 +36,7 @@
 	),
 	'extensions' => array(
 		'already_enabled' => '%s is already enabled',
+		'cannot_delete' => '%s cannot be deleted',
```
Comment (Frenzie) [inline app/i18n/en/feedback.php]: I left the comment only on one line but I did mean it for both. ;-)

Comment (aledeg) [reply app/i18n/en/feedback.php:39]: What do you have in mind for that one?

**Review by Frenzie — APPROVED** [review]
lgtm :-)

**Pull request merged**